ccdproc/image_collection.py

```diff
@@ -58,14 +58,19 @@ class ImageFileCollection(object):
         The filenames are assumed to be in ``location``.
         Default is ``None``.
 
+     hdu: str or int, optional
+         Name/Index of the extension, which specifies the extension to extract data
+         from all files in a given directory.
+         Default is 0.
+
```

Comment:
Could you add a `.. versionadded: 1.3` here? Needs a newline before and after and needs to have the same indentation as the descritpion.